### DISCUSSION TITLE
Auto-generate dates in the WorkBench

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5797] [Feature Requests]
> We’ve realized that creating cataloged date is not automatic in Specify7 WorkBench.
> In Sp6 if I uploaded a dataset without cataloged date, it was set to the current date by default. In Sp7 it stays blank (NULL value).
> Is there a way for automatic generation as with the catalog numbers? ([Sp7 WorkBench: Auto-insert Catalog Numbers](https://discourse.specifysoftware.org/t/sp7-workbench-auto-insert-catalog-numbers/1150))
> Thanks for your help!

**Requested By:** [@ZsPapp](https://discourse.specifysoftware.org/t/sp7-workbench-auto-insert-cataloged-date/1163) on the Speciforum on behalf of NHMD

Per my conversation with @maxpatiiuk, we need to look at the Specify 6 source code to see all the fields for which it has a default value **AND** then check if that list makes sense or if Specify 7 should have default dates for different/additional fields.

**Comment by specifysoftware** [comment]
This issue has been mentioned on **Specify Community Forum**. There might be relevant details there:

https://discourse.specifysoftware.org/t/sp7-workbench-auto-insert-cataloged-date/1163/2

**Comment by totallynotleo** [comment]
Good morning, 
I've just come across this feature discussion whilst looking at moving some workbench stuff from Specify 6 to Specify 7 and was wondering if there have been any updates on auto-generating workbench dates? 
If not, can I also be added as a requester of this feature on behalf of the Queensland Herbarium? Thanks!
